src/containers/Masthead/components/MastheadSearch.tsx

```diff
@@ -142,6 +146,21 @@ export const MastheadSearch = () => {
     }
   };
 
+  const handleContextId = async (urlId: string) => {
+    try {
+      const nodes = await fetchNodes({
+        contextId: urlId,
+        language: i18n.language,
+        taxonomyVersion,
+      });
+      const arr = nodes[0]?.contentUri?.split(":") ?? [];
+      const id = arr[arr.length - 1];
+      navigate(routes.editArticle(parseInt(id), "topic-article"));
```

Comment:
Er det gitt at de skal sendes til topic-article?

Reply:
nei!

Reply:
Kan vi isåfall utlede hvor vi skal sende de fra nodetypen?

Reply:
Endrer til standard. Vi har aller flest standardartikler. Og du blir uansett redirigert til korrekt skjema dersom du sendes feil. Dette gjør at antall feilredigeringer er minimalt.